src/swarm/util/RecordBatcher.d

```diff
@@ -590,3 +590,60 @@ public class RecordBatch : RecordBatchBase
         return value;
     }
 }
+
+version ( UnitTest )
+{
+    import ocean.core.Test;
+}
+
+// Basic tests.
+unittest
+{
+    auto lzo = new Lzo;
+    auto writer = new RecordBatcher(lzo, 200);
+    auto reader = new RecordBatch(lzo, 200);
+
+    mstring key, value_S, value_M, value_L;
+    key.length = 16;
+    value_S.length = 50;
+    value_M.length = 150;
+    value_L.length = 200;
+
+    // Test RecordBatcher.batchedSize()
+    test!("==")(writer.batchedSize(key, value_S), 16 + 8 + 50 + 8);
```

Comment:
The numbers are probably `key.length.sizeof + key.length + value_S.length.sizeof + value_S.length`, maybe make it explicit.

Reply:
Yeah true.

Reply:
Updated.